Prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+"""CP1404 | prac_06 guitars| Darcy Kemp
+program to create guitar objects from user input and append them to a list
+
+I have no idea where to put the estimation, I will put it here.
+
+est: 40
+real: 45
+"""
+from Prac_06.guitar import Guitar
+
+guitars = []
+
+guitar_name = "-1"
+
+while guitar_name != "":
```

Comment:
this while loop doesn't follow the standard loop pattern, it will work fine. However Lindsey would probable prefer you to use the loop pattern:[ something = input(), while something != "": do things and the last line of the loop is the same as the last line before the loop opens]